--- rsxs-1.0/lib/argp-parse.c.str_fmt~	2014-04-09 14:53:41.720922930 +0200
+++ rsxs-1.0/lib/argp-parse.c	2014-04-09 14:53:43.989764735 +0200
@@ -154,7 +154,7 @@ argp_version_parser (int key, char *arg,
       else if (argp_program_version)
 	fprintf (state->out_stream, "%s\n", argp_program_version);
       else
-	__argp_error (state, dgettext (state->root_argp->argp_domain,
+	__argp_error (state, "%s", dgettext (state->root_argp->argp_domain,
 				       "(PROGRAM ERROR) No version known!?"));
       if (! (state->flags & ARGP_NO_EXIT))
 	exit (0);
